src/renderer/store/tachyon.store.ts

```diff
@@ -40,6 +67,221 @@ async function connect() {
     }
 }
 
+async function subscribeList() {
+    try {
+        tachyonStore.error = undefined;
+        const response = await window.tachyon.request("lobby/subscribeList");
+        //Per Tachyon protocol, this subscribes us, but does not return an updated list, that happens in the ListUpdated or ListReset events.
+        console.log("subscribeList:", response.status);
+    } catch (error) {
+        console.error("Error with request lobby/subscribeList:", error);
+        tachyonStore.error = "Error with request lobby/subscribeList";
+        tachyonStore.lobbyList = {};
+    }
+}
+
+async function unsubscribeList() {
+    try {
+        tachyonStore.error = undefined;
+        const response = await window.tachyon.request("lobby/unsubscribeList");
+        console.log("Tachyon: lobby/unsubscribeList:", response.status);
+    } catch (error) {
+        console.error("Error with request lobby/unsubscribeList:", error);
+        tachyonStore.error = "Error with request lobby/unsubscribeList";
+    }
+}
+
+async function createLobby(data: LobbyCreateRequestData) {
+    try {
+        battleActions.resetToDefaultBattle(undefined, undefined, undefined, true);
+        battleStore.isOnline = true;
+        battleStore.isLobbyOpened = true;
```

Comment:
This will open the BattleDrawer even if the request is not successful. This line should be after the request is made so it does not open when it should not (e.g. on failure).